test/jest/Corporation.test.ts

```diff
@@ -112,4 +116,59 @@ describe("Corporation", () => {
       expectSharesToAddUp(corporation);
     });
   });
+
+  describe("helpers.calculateOfficeSizeUpgradeCost", () => {
+    // These values were pulled from v2.6.0 as reference test values
+    const refCosts = {
+      from3: {
+        3: 4360000000,
+        15: 26093338259.60001,
+        150: 3553764305895.2593,
+      },
+      from6: {
+        3: 4752400000.000001,
+        15: 28441738702.964012,
+        150: 3873603093425.833,
+      },
+      from9: {
+        3: 5180116000.000002,
+        15: 31001495186.230778,
+        150: 4222227371834.1587,
+      },
+    };
+
+    describe("upgrade office size from size 3", () => {
```

Comment:
You can use table-driven tests to take all the redundancy out of this; you've already got most of the data structure set up, above.

Reply:
I was previously unaware of table-driven tests.  Thanks; this is way cleaner now.